package.json

```diff
@@ -1,269 +1,253 @@
-    "style-loader": "^0.23.0",
-    "stylelint": "^9.6.0",
-    "stylelint-config-standard": "^18.2.0",
-    "stylelint-order": "^1.0.0",
-    "tabbable": "^1.1.3",
-    "webpack": "^4.19.0",
-    "webpack-bundle-analyzer": "^3.0.2",
-    "webpack-cli": "^3.1.0",
-    "webpack-dev-server": "^3.1.8",
-    "worker-farm": "^1.6.0"
-  },
-  "peerDependencies": {
-    "axios": "^0.18.0",
-    "box-react-ui": "^26.17.0",
-    "classnames": "^2.2.5",
-    "draft-js": "^0.10.1",
-    "filesize": "^3.6.0",
-    "form-serialize": "^0.7.2",
-    "immutable": "^3.7.4",
-    "jsuri": "^1.3.1",
-    "lodash": "^4.17.5",
-    "pikaday": "^1.6.1",
-    "react": "^16.5.0",
-    "react-animate-height": "^2.0.4",
-    "react-dom": "^16.5.0",
-    "react-immutable-proptypes": "^2.1.0",
-    "react-intl": "^2.3.0",
-    "react-measure": "^2.1.2",
-    "react-modal": "^3.5.1",
-    "react-process-string": "^1.2.0",
-    "react-tether": "^0.6.1",
-    "react-textarea-autosize": "^6.1.0",
-    "react-virtualized": "^9.20.1",
-    "regenerator-runtime": "^0.11.1",
-    "scroll-into-view-if-needed": "^1.5.0",
-    "tabbable": "^1.1.2"
-  },
-  "dependencies": {}
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/box/box-ui-elements.git"
+    },
+    "bugs": {
+        "url": "https://github.com/box/box-ui-elements/issues"
+    },
+    "scripts": {
+        "build": "yarn run pre-build && LANGUAGE=en-US BABEL_ENV=dev NODE_ENV=dev webpack --progress --colors --config build/webpack.config.js --mode development",
+        "build-ci": "LANGUAGE=en-US REACT=true yarn run build-prod && LANGUAGE=en-US REACT=false yarn run build-prod",
+        "build:es": "./node_modules/.bin/babel src -d lib",
+        "build-prod": "NODE_ENV=production webpack --config build/webpack.config.js --mode production",
+        "build-npm": "yarn run npm && yarn run build-lib",
+        "build-link": "yarn run npm && yarn run build-link-lib",
+        "build:link-dev": "yarn run npm && yarn run build-link-lib -w",
+        "build-link-lib": "yarn run clean:lib && BABEL_ENV=dev NODE_ENV=dev yarn run build:es --ignore \"**/__tests__/**\",\"src/components/messages.js\" --copy-files -s inline",
+        "build-lib": "yarn run clean:lib && BABEL_ENV=npm NODE_ENV=production yarn run build:es --ignore \"**/__tests__/**\"",
+        "build:i18n": "props2es",
+        "changelog": "./node_modules/.bin/conventional-changelog -i CHANGELOG.md -s -r 0",
+        "clean": "rm -rf dist lib reports/* i18n/json i18n/*.js styleguide",
+        "clean:lib": "rm -rf lib",
+        "clean-styleguide": "rm -rf styleguide",
+        "debug": "NODE_ENV=test yarn run jest --watch",
+        "dev": "yarn run pre-build && LANGUAGE=en-US BABEL_ENV=dev NODE_ENV=dev node --max_old_space_size=8192 node_modules/webpack-dev-server/bin/webpack-dev-server.js --config build/webpack.config.js --mode development",
+        "github-release": "./node_modules/.bin/conventional-github-releaser",
```

Comment:
`"github-release": "./node_modules/.bin/conventional-github-releaser -p angular",`

Reply:
isn't moving to semantic-release in a separate PR?